### PR TITLE
Create database

### LINKED ISSUE #30: [2pt] - Create database
- Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
    - music_albums table (add all properties and associations from the parent Item class as table columns)
    - genres table

### PR DESCRIPTION
**Pull request by Yazino12**
### In this pull:

- Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
    - music_albums table (add all properties and associations from the parent Item class as table columns)
    - genres table

**This closes #30**

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -20,3 +20,24 @@ CREATE TABLE books (
   CONSTRAINT fk_source_id FORIEGN KEY (source_id) REFERENCES sources(id),
   CONSTRAINT fk_label_id FORIEGN KEY (label_id) REFERENCES labels(id)
 );
+
+CREATE TABLE geners(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR,
```
Comment (aimalamiri) [inline schema.sql]: Remove the comma at the end of this line.

File: schema.sql [hunk 2]
```diff
@@ -20,3 +20,24 @@ CREATE TABLE books (
...
+  author_id INT,
+  source_id INT,
+  label_id INT,
+  publish_date DATE,
+  archived BOOLEAN,
+  on_spotify BOOLEAN,
+  CONSTRAINT fk_genre_id FORIEGN KEY (genre_id) REFERENCES genres(id),
+  CONSTRAINT fk_author_id FORIEGN KEY (author_id) REFERENCES authors(id),
+  CONSTRAINT fk_source_id FORIEGN KEY (source_id) REFERENCES sources(id),
+  CONSTRAINT fk_label_id FORIEGN KEY (label_id) REFERENCES labels(id),
```
Comment (aimalamiri) [inline schema.sql]: And also please Remove the comma at the end of this line, which will cause an error.

**Review by aimalamiri — CHANGES_REQUESTED** [review]
## Almost there

### Status: Changes required

Well done @Yazino12 you are almost there. Please check the comments below for some changes.

File: schema.sql (hunk 2 above)
Comment (Yazino12) [reply schema.sql]: @aimalamiri, I will tackle these issues right away 😇.

**Review by aimalamiri — APPROVED** [review]
**Hello @Yazino12 👋🏾**

### PROJECT STATUS: APPROVED 🟢🟢🟢

**To recap:**

- No linter errors ✔
- Data preserved correctly ✔
- Good design ✔



Congratulations on successfully completing this task 🎉🎉🍾🍾. There is nothing to say, but it's time to merge it 🚀.

**Pull request merged**